README.md

```diff
@@ -59,58 +60,185 @@ The connectors marked with :x: are known as currently broken.
+|:---------|:--------------------|:-----------|:----------------------------------------------------------------------|
+| Github   | Commits             | doubleface | :white_check_mark:                                                    |
+| Twitter  | Published tweets    | doubleface | :white_check_mark:                                                    |
+| Linkedin | contact information | Peltoche   | :x: ([#396](https://github.com/cozy-labs/konnectors/issues/396))      |
+| Google   | contact information | jacquarg   | :warning:([#541](https://github.com/cozy-labs/konnectors/issues/541)) |
 
-## Events
+### Events
 
 | Website           | Data            | Maintainer     | Status                                                                 |
 |:------------------|:----------------|:---------------|:-----------------------------------------------------------------------|
-| Ical Feed         | Events          | Cozy              | :white_check_mark:                                                     |
+| Ical Feed         | Events          | Cozy           | :white_check_mark:                                                     |
 | Contact Birthdays | Events          | Frank Rousseau | :white_check_mark:                                                     |
 | Doctolib          | Events          | ZeHiro         | :white_check_mark:                                                     |
 | Facebook          | Events          | jacquarg       | :warning: ([#420](https://github.com/cozy-labs/konnectors/issues/420)) |
 | SNCF              | Events          | Chocobozzz     | :white_check_mark:                                                     |
-| Google            | Events          | Cozy              | :white_check_mark:                                                     |
+| Google            | Events          | Cozy           | :white_check_mark:                                                     |
 | Isen              | Events, lessons | Cozy           | :white_check_mark:                                                     |
 
-## Productivity
+### Productivity
 
-| Website    | Data      | Maintainer | Status             |
-|:-----------|:----------|:-----------|:-------------------|
-| RescueTime | Activites | looking for a maintainer with credentials          | :white_check_mark: |
+| Website    | Data      | Maintainer                                | Status             |
+|:-----------|:----------|:------------------------------------------|:-------------------|
+| RescueTime | Activites | looking for a maintainer with credentials | :white_check_mark: |
 
-## Hobbies
+### Hobbies
 
 | Website  | Data                    | Maintainer | Status             |
 |:---------|:------------------------|:-----------|:-------------------|
 | Podcasts | Audio podcasts episodes | Babolivier | :white_check_mark: |
 
-# Build
 
-To build this application:
+Hack
+----
+
+### Install and run in dev mode
```

Comment:
Yep. We try to have the README as fresh as possible. Maybe we should avoid to put redudant information in the wiki and keep it only for specific cases, internal to the project.